specification/cognitiveservices/data-plane/FormRecognizer/preview/v2.0/FormRecognizer.json

```diff
@@ -490,6 +453,100 @@
       }
     }
   },
+  "x-ms-paths": {
+    "/custom/models?op=full": {
+      "get": {
+        "summary": "List Custom Models",
+        "description": "Get information about all custom models",
+        "operationId": "ListCustomModels",
+        "consumes": [],
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "name": "op",
+            "in": "query",
+            "description": "Specify whether to return summary or full list of models.",
+            "required": false,
+            "default": "full",
+            "type": "string",
+            "enum": [
+              "full",
+              "summary"
+            ]
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Success",
+            "schema": {
+              "$ref": "#/definitions/Models"
+            }
+          },
+          "default": {
+            "description": "Response entity accompanying non-successful responses containing additional details about the error.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink",
+          "itemName": "modelList"
+        },
+        "x-ms-examples": {
+          "List custom models": {
+            "$ref": "./examples/GetModels.json"
+          }
+        }
+      }
+    },
+    "/custom/models?op=summary": {
+      "get": {
+        "summary": "Get Custom Models",
+        "description": "Get information about all custom models",
+        "operationId": "GetCustomModels",
+        "consumes": [],
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "name": "op",
```

Comment:
Should be required with a single enum value "summary":
https://github.com/Azure/azure-rest-api-specs/blob/afe2cdb70a548ebf0348e433bf7d0437a2a8c986/specification/storage/data-plane/Microsoft.BlobStorage/preview/2019-12-12/blob.json#L161-L168